common.sh

```diff
@@ -418,6 +418,15 @@ script_builder()
     fi
 
     ${label}_install_deps
+    # install CUDA toolkit only for non-gdr test on x86_64 platform.
+    if [ "$ami_arch" = "x86_64" ] && [ "$BUILD_GDR" -eq 0 ]; then
+        cat <<-"EOF" >> ${tmp_script}
+        curl -O https://developer.download.nvidia.com/compute/cuda/11.0.3/local_installers/cuda_11.0.3_450.51.06_linux.run
+        chmod +x cuda_11.0.3_450.51.06_linux.run
+        sudo ./cuda_11.0.3_450.51.06_linux.run --silent --toolkit
```

Comment:
Does the return code need to be checked or is there a `set -e` hiding somewhere?

Reply:
There is a `set -e` hiding in set_var in script_builder: https://github.com/awslabs/libfabric-ci-scripts/blob/master/common.sh#L518-L529. So if there is an error here the script will fail and exit.